promises/Task2.js

```diff
@@ -0,0 +1,54 @@
+// Завдання 2
+// Напишіть функцію яке буде робити запит до серверу за цим ендпоінтом
+// https://jsonplaceholder.typicode.com/todos/1
+// Функція повинна повертати як результат Promise що повертає об’єкт todo
+
+const url1 = "https://jsonplaceholder.typicode.com/todos/1"
+function fetchToDo(url1) {
```

Comment:
```js
function fetchToDo(url1) {
    return fetch(url1)
        .then((response) => response.json())
}
```

Reply:
.then((jsonBody) => jsonBody) - this does not make sence

Reply:
remember about naming 
Response - bad
response -good

Only classes starts with upper letter